src/test/java/uk/gov/digital/ho/pttg/api/RequestHeaderDataTest.java

```diff
@@ -63,89 +78,115 @@ public void shouldDefaultRequestData() {
     }
 
     @Test
     public void shouldUseUserIdFromRequest() {
-        when(mockHttpServletRequest.getHeader("x-auth-userid")).thenReturn("some user id");
+        given(mockHttpServletRequest.getHeader("x-auth-userid"))
+                .willReturn("some user id");
 
-        requestData.preHandle(mockHttpServletRequest, mockHttpServletResponse, mockHandler);
+        when_requestDataPrehandleCalled();
 
         assertThat(requestData.userId()).isEqualTo("some user id");
     }
 
     @Test
     public void shouldDefaultMaxDurationWhenNotSuppliedInRequest() {
-        when(mockHttpServletRequest.getHeader("x-max-duration")).thenReturn(null);
+        given(mockHttpServletRequest.getHeader("x-max-duration"))
+                .willReturn(null);
 
-        requestData.preHandle(mockHttpServletRequest, mockHttpServletResponse, mockHandler);
+        when_requestDataPrehandleCalled();
 
-        assertThat(MDC.get("max_duration")).isEqualTo("60000");
+        // This is not a Spring test, so the default value is zero
+        assertThat(MDC.get("max_duration")).isEqualTo("0");
     }
 
     @Test
     public void shouldThrowExceptionWhenNonIntegerMaxDuration() {
-        when(mockHttpServletRequest.getHeader("x-max-duration")).thenReturn("a76543");
+        given(mockHttpServletRequest.getHeader("x-max-duration"))
+                .willReturn("a76543");
 
-        assertThatThrownBy(() -> requestData.preHandle(mockHttpServletRequest, mockHttpServletResponse, mockHandler))
-                .isInstanceOf(IllegalArgumentException.class);
+        assertThatExceptionOfType(IllegalArgumentException.class)
+            .isThrownBy(() -> requestData.preHandle(mockHttpServletRequest, mockHttpServletResponse, mockHandler));
 
         assertThat(MDC.get("max_duration")).isEqualTo(null);
     }
 
     @Test
     public void shouldUseMaxDurationFromRequest() {
-        when(mockHttpServletRequest.getHeader("x-max-duration")).thenReturn("76543");
+        given(mockHttpServletRequest.getHeader("x-max-duration"))
+                .willReturn("76543");
 
-        requestData.preHandle(mockHttpServletRequest, mockHttpServletResponse, mockHandler);
+        when_requestDataPrehandleCalled();
 
         assertThat(MDC.get("max_duration")).isEqualTo("76543");
     }
 
     @Test
     public void shouldLogWhenCorrelationIdGenerated() {
-        when(mockHttpServletRequest.getHeader("x-correlation-id")).thenReturn(null);
+        given(mockHttpServletRequest.getHeader("x-correlation-id"))
+                .willReturn(null);
 
-        requestData.preHandle(mockHttpServletRequest, mockHttpServletResponse, mockHandler);
+        when_requestDataPrehandleCalled();
 
-        verify(mockAppender).doAppend(argThat(argument -> {
-            LoggingEvent loggingEvent = (LoggingEvent) argument;
+        then(mockAppender)
+                .should()
+                .doAppend(argThat(argument -> {
```

Comment:
Changing to the then.should() format has brought the indentation in even further here and it's now quite ugly. Could the lambda perhaps be extracted to a method?